src/pages/form/step-form/locales/en-US.ts

```diff
@@ -0,0 +1,62 @@
+export default {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)